content/en/platform/corda/4.11/community/release-notes.md

```diff
@@ -105,6 +105,15 @@ This release includes the following fixes:
 
 The following database changes have been applied:
 
+* The `vault_state` table now includes a `consuming_tx_id` column. The new column was added in the following migration script: `vault-schema.changelog-v14.xml`.
+
+* Two Phase Finality introduces an additional data field within the main `DbTransaction` table:
+
+```kotlin
```

Comment:
This code block should be indented in line with the bullet that it relates to.

Reply:
Indented the code.